src/content/docs/en/guides/astro-db.mdx

```diff
@@ -206,17 +384,17 @@ This example inserts a row into a `Comment` table based on a parsed form POST re
 import { db, Comment } from 'astro:db';
 
 if (Astro.request.method === 'POST') {
-  // parse form data
+  // Parse form data
   const formData = await Astro.request.formData();
   const author = formData.get('author');
   const content = formData.get('content');
```

Comment:
The rest of the page uses `body` for the comments while these few examples use `content`

I suggest making everything uniform

```suggestion
  const body = formData.get('body');
```